### ISSUE TITLE
About all_text_embedding

### ISSUE THREAD

**Issue by LeslieZJC** [ictnlp/LLaVA-Mini#27]
Dear Author, while reviewing the code, I noticed the following issue:`all_text_embedding = self.get_input_embeddings()(input_ids.clamp(min=0)).detach()`This all_text_embedding includes embeddings for the ​​answer​​ (e.g., GPT output), which are then fed into prefusion_layers for fusion with other features. Is this design correct?
